scripts/update_terminfo.sh

```diff
@@ -72,18 +76,19 @@ EOF
 
 for term in $sorted_terms; do
   path="$(find "$db" -name "$term")"
-  if [[ -z $path ]]; then
-    echo "Not found: $term. Skipping." 1>&2
+  if [ -z $path ]; then
```

Comment:
If you're removing the `[[` bashism (even though it's a bash script), then you need to quote `$path`.

```suggestion
  if [ -z "$path" ]; then
```